sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -407,8 +422,8 @@ class HealthcareEntity(DictMixin):
         in the original request, which is UnicodeCodePoints by default.
     :ivar float confidence_score: Confidence score between 0 and 1 of the extracted
             entity.
-    :ivar links: A collection of entity references in known data sources.
-    :vartype links: list[~azure.ai.textanalytics.HealthcareEntityLink]
+        :ivar data_sources: A collection of entity references in known data sources.
+        :vartype data_sources: list[~azure.ai.textanalytics.HealthcareEntityDataSource]
```

Comment:
there seems to be incorrect indents in the docstirng

Reply:
Fixed!